src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/Tasks/DoNotCreateTaskCompletionSourceWithWrongArguments.cs

```diff
@@ -0,0 +1,54 @@
+// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Collections.Immutable;
+using Analyzer.Utilities;
+using Analyzer.Utilities.Extensions;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.Diagnostics;
+using Microsoft.CodeAnalysis.Operations;
+
+namespace Microsoft.NetCore.Analyzers.Tasks
+{
+    /// <summary>CA2012: Use ValueTasks correctly.</summary>
+    [DiagnosticAnalyzer(LanguageNames.CSharp, LanguageNames.VisualBasic)]
+    public sealed class DoNotCreateTaskCompletionSourceWithWrongArguments : DiagnosticAnalyzer
+    {
+        internal const string RuleId = "CA2247";
```

Comment:
Seems like I accidentally marked `CA2247` as a used ID: https://github.com/dotnet/roslyn-analyzers/blob/6b16c9c25464f5f77d33da7a62f803857d4d3d6a/src/Utilities/Compiler/DiagnosticCategoryAndIdRanges.txt#L17

It is not used, and you don't need to update the range.

Reply:
Ok, thanks.